schemas/content/product.schema.json

```diff
@@ -37,21 +37,14 @@
           "type": "string",
           "description": "The localized description of the product."
         },
-        "xdm:category": {
+        "xdm:categories": {
           "title": "Category",
-          "type": "string",
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
This is hard to resolve. If your IDs are plain strings, you have a black-box API that does not allow resolution of metadata of a category.